app/app/Http/Controllers/LocationSearchController.php

```diff
@@ -400,4 +400,16 @@ public function search(Request $request)
 				'search_radius'      => $search_radius
 			]);
 	}
+	/**
+	 * set search radius
+	 */
+	public function setSearchRadius(Request $request) {
+		$distance = Input::get('distance');
+		if(is_numeric($distance)) {
```

Comment:
Also, several of your commit messages talk of "issue 60" but this is issue 260.  Saying "issue 260" would be good for any developers reading the message but #260 gets interpretted by github as being linked to the issue and will show in the comments under the issue.  You could correct this by doing an interactive( -i ) rebase, marking the commits with "edit" instead of "pick" and amending( --amend ) the messages for each of the commits before continuing the rebase to its successful end.  I'll do this for you if this is too difficult or you prefer.